test/sails.sockets.test.js

```diff
@@ -262,6 +262,11 @@ describe('low-level socket methods:', function (){
         });
       });
     });
+    it('should warn and not error if passing in null socket', function(done) {
+      // Testing fix for #6
+      sails.sockets.leave(null);
```

Comment:
This looks fine--we generally don't have tests for log messages.  As long as it tests that the faulty behavior was fixed, it's good!